app/settings.py

```diff
@@ -263,6 +264,18 @@
 # Maximum file upload size for forms
 MAX_UPLOAD_SIZE = 5242880
 
+# Max number of votation application from 1 to MAX_VOTES
 MAX_VOTES = 10
 
+# Max number of votes that can have one application
 MAX_VOTES_TO_APP = 50
+
+# Mentor/Volunteer applications can expire if they are invited, set to False to not
+MENTOR_EXPIRES = False
+VOLUNTEER_EXPIRES = False
+
```

Comment:
This 2 go to hackathon variables. `APPLICATION_EXPIRATION_TYPES` stays in settings.py